src/odbc_writer.rs

```diff
@@ -36,7 +36,7 @@ mod boolean;
 mod map_arrow_to_odbc;
 mod text;
 mod timestamp;
-
+const PROTECTED_KEYWORDS_LOWER: &[&str] = &["index", "key", "values"];
```

Comment:
Please make this constant local to the `quote_column_name` function. In addition I would suggest renaming it to `RESERVED_KEYWORDS`.